### ISSUE TITLE
More than 4 from Wellcome Library

### ISSUE THREAD

**Issue by jennpb** [LibraryCarpentry/queriesquestions#1]
Hi @drjwbaker 
I've now got another four people interested in LIbrary Carpentry. Do you think you can manage 5 from Wellcome Library, or should I pull names from a hat?
Thanks!

**Comment by drjwbaker** [comment]
Hi @jennpb. I hope we can accommodate all 5, but if we have no drop-outs from the original expressions of interest it is looking at little tight both from a space and crowd control perspective. Would you be happy with 4 + 1 reserve for now?

**Comment by jennpb** [comment]
Certainly. I'll let them know, and send the names of the 3 others as soon as I can.
